src/components/Questions.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Questions } from './Questions'
+
+test('show the timer', () => {
+    render(<Questions />)
+
+    expect(screen.getByText('16:40')).toBeInTheDocument()
+})
+
+test('show the score', () => {
+    render(<Questions />)
+
+    expect(screen.getByText('000000')).toBeInTheDocument()
+})
+
+test('show the question', () => {
+    // How to mock getAllSelectedCategories()? I could pass them as props
```

Comment:
@enpayne Any idea how I could mock those functions in the test? I saw I can do it with enzyme's shallow, but is there another way?